src/dynarec/la64/dynarec_la64_0f.c

```diff
@@ -491,6 +491,37 @@ uintptr_t dynarec64_0F(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
             }
             if (!rex.w) ZEROUP(gd);
             break;
+        case 0xBF:
+            INST_NAME("MOVSX Gd, Ew");
+            nextop = F8;
+            GETGD;
+            if (MODREG) {
+                ed = TO_LA64((nextop & 7) + (rex.b << 3));
+                EXT_W_H(gd, ed);
+            } else {
+                SMREAD();
+                addr = geted(dyn, addr, ninst, nextop, &ed, x3, x1, &fixedaddress, rex, NULL, 1, 0);
+                LD_H(gd, ed, fixedaddress);
+            }
+            if (!rex.w) ZEROUP(gd);
+            break;
+        case 0xC6:
+            INST_NAME("SHUFPS Gx, Ex, Ib");
+            nextop = F8;
+            GETGX(v0, 1);
+            if (MODREG) {
+                v1 = sse_get_reg(dyn, ninst, x1, (nextop & 7) + (rex.b << 3), 0);
+            } else {
+                addr = geted(dyn, addr, ninst, nextop, &wback, x2, x3, &fixedaddress, rex, NULL, 1, 1);
+                v1 = fpu_get_scratch(dyn);
+                VLD(v1, ed, fixedaddress);
+            }
+            u8 = F8;
+            q0 = fpu_get_scratch(dyn);
+            VOR_V(q0, v1, v1);
```

Comment:
those 2 lines are probably useless if GX==EX.
But if GX!=EX, q0 is probably not usefull and stuff can be done in v0 directly?

Reply:
Oh I missed the GX!=EX suggestion, sorry.

Reply:
Okay done, much better, thanks!